### PR TITLE
Store API Listing

### PR DESCRIPTION
**Pull request by kavishkafernando**
## Purpose
 API Listing in Store

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js [hunk 1]
```diff
@@ -166,10 +177,10 @@ export default class ProtectedApp extends Component {
                 </IntlProvider>
             );
         }
-
+
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js]: do we need this line?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js [hunk 2]
```diff
@@ -166,10 +177,10 @@ export default class ProtectedApp extends Component {
                 </IntlProvider>
             );
         }
-
+    
         let params = qs.stringify({referrer: this.props.location.pathname});
         return (
             <Redirect to={{pathname: '/login', search: params}}/>
-        );
+        );
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js]: also this line?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js]: done

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/ProtectedApp.js]: done

**Pull request merged**